fiftyone/utils/clip/zoo.py

```diff
@@ -139,13 +163,15 @@ def _prepare_text_features(self):
 
             text_features[i, : len(tokens)] = torch.tensor(tokens)
 
-        return text_features
+        with torch.no_grad():
+            return self._model.encode_text(text_features)
```

Comment:
Can we just use `encode_text` like the model already does? For users who already have worked with text models previously, `encode_text` immediately allows them to know what the method is doing and is even guessable.